Audio files and scripts

DanielKelowna opened the pull request:
Not final implementation, need to make game object to attach it to. Unclear as to what scene/level

No idea what happened to main menu. Review changes before approving. If necessary, make backup, approve, then overwrite

DanielKelowna committed:
Audio files and scripts

Not final implementation, need to make game object to attach it to. Unclear as to what scene/level

ivonanicetin reviewed (changes requested):
Not sure why there is changes in the MainMenu, I don't think there should be any.
I tested the game and there is a "NullReferenceException" error. 
The ball is not behaving as intended, it doesn't bounce off the the slider when it comes back, ball just sticks.

DanielKelowna committed:
Functional

Audio implemented. Tested and working. Need UI fix. Can apply offset to ball to prevent it from getting "stuck"
